test/simplifyExpression/oneStep.test.js

```diff
@@ -96,3 +96,15 @@ describe('simplifyDoubleUnaryMinus step actually happens', function () {
     assert.equal(steps[0].changeType, ChangeTypes.RESOLVE_DOUBLE_MINUS);
   });
 });
+
+describe('division by zero', function () {
```

Comment:
can you name this 'division by zero error' just to be super clear when tests are running?